tests/test_moduletool.py

```diff
@@ -667,6 +667,34 @@ def test_project_freeze(modules_dir, modules_repo, capsys):
     )
 
 
+def test_project_freeze_disk(modules_dir, modules_repo, capsys):
+    coroot = install_project(modules_dir, "modA")
```

Comment:
To me it's quite confusing that modA is the name of a project. Changing this to projectA would improve the readability of the code. modA is a dependency of several test cases. Not sure if we should handle this in the current PR.

Reply:
+1